src/hotspot/share/gc/shenandoah/shenandoahGeneration.hpp

```diff
@@ -52,6 +52,7 @@ class ShenandoahGeneration : public CHeapObj<mtGC> {
 protected:
   // Usage
   size_t _affiliated_region_count;
+  size_t _humongous_waste;      // how much waste for padding in humongous objects
```

Comment:
Could you document here whether this is included in `_used` below, or is explicitly excluded from it. From my cursory reading of the code doing the adjustments, it looked like `_used` includes `_humongous_waste`.

it would also be good to include a line of documentation about where this statistic might be useful (or, if it is, where it is used today).

Reply:
It isn't included in `used`, but is deducted from `available`. This gives the heuristics a more accurate view of how much memory remains for allocations. I will update the comment.

Reply:
Makes sense. Thanks for updating the documentation.